infra/ansible/remote/nodes/playbooks/start/metagraph-l0/genesis.ansible.yml

```diff
@@ -69,6 +69,17 @@
     msg: "File /home/{{ ansible_user }}/code/metagraph-l0/{{ owner_p12_file_name }} does not exist"
   when: not (owner_p12_file_name is undefined or owner_p12_file_name == "")  and not owner_file.stat.exists
 
+- name: Check if additional owner file exists
```

Comment:
What's the additional owner file? Isn't there just one file that represents the owner?